Lindhart.Analyser.MissingAwaitWarning/Lindhart.Analyser.MissingAwaitWarning/Lindhart.Analyser.MissingAwaitWarning/Resources.resx

```diff
@@ -129,6 +129,10 @@
     <value>Possible missing await keyword</value>
     <comment>The title of the diagnostic.</comment>
   </data>
+  <data name="StrictAnalyzerMessageFormat" xml:space="preserve">
+    <value>The async method '{0}' returns a Task that is not awaited. This could be the expected behaviour, depending of your context.</value>
```

Comment:
We don't investigate if this `Task` is not awaited. We only investigate if it is awaited on the current line. I believe this should be explicit in the message. The `Task` can very well be awaited on another line (the most used use-case I've seen for assigning the `Task` to a variable).